compiler/passes/codegen.cpp

```diff
@@ -360,6 +361,38 @@ compareSymbol(const void* v1, const void* v2) {
   return result;
 }
 
+static int
+compareSymbol2(void* v1, void* v2) {
+  Symbol* s1 = (Symbol*)v1;
+  Symbol* s2 = (Symbol*)v2;
+  ModuleSymbol* m1 = s1->getModule();
+  ModuleSymbol* m2 = s2->getModule();
+  if (m1 != m2) {
```

Comment:
You're right, that was an instance of going off my memory of past results rather than what the function description actually defined, sorry.

I'll re-examine the if/else statement.